src/index.html

```diff
@@ -7,8 +7,51 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <section class="logo">
+        <a href="#Home">
+          <img src="../src/images/logo.png" alt="logo-moyo">
+        </a>
+      </section>
+      <nav class="nav">
+        <ul class="nav_ul">
+          <li>
+            <a href="#apple" class="nav___link is-active">APPLE</a>
```

Comment:
![image](https://user-images.githubusercontent.com/98764641/199216355-4f3d76da-b211-49d0-b9a8-2f4c7068363c.png)

Reply:
i have a uppercase in css but also i have this, sorry. fixed.